extensions/amp-story-360/0.1/amp-story-360.js

```diff
@@ -81,7 +80,21 @@ const buildActivateButtonTemplate = (element) => {
       </span>
     </button>
   `;
-};
+
+/**
+ * Generates the template for the gyroscope feature discovery animation.
+ *
+ * @param {!Element} element
+ * @return {!Element}
+ */
+const buildDiscoveryTemplate = (element) => htmlFor(element)`
+    <div class="i-amphtml-story-360-discovery">
```

Comment:
nit: can we give a role=dialog and aria-labelledby to that div?

Reply:
Thanks for identifying an opportunity for a11y :) 

Since this div disappears I think we want it to announce it when it's displayed but not be user selectable. 
I found adding `aria-live="polite"` to the parent and `aria-hidden="true"` to the text will accomplish this.